upf/upf_pfcp.c

```diff
@@ -2058,8 +2092,9 @@ pfcp_lookup (uint64_t sess_id)
 }
 
 static int
-urr_increment_and_check_counter (u64 * packets, u64 * bytes, u64 * consumed,
-				 u64 threshold, u64 quota, u64 n_bytes)
+urr_increment_and_check_counter (u64 * packets, u64 * bytes,
+				 u64 * consumed, u64 threshold, u64 quota,
+				 u64 n_bytes)
```

Comment:
pure white space and formatting changes should not go into this PR.

There a are more in here, but I'm not going to point out each one individually.